Mathlib/MeasureTheory/Function/L1Space/Integrable.lean

```diff
@@ -407,10 +500,16 @@ theorem Integrable.sub {f g : α → β} (hf : Integrable f μ) (hg : Integrable
 theorem Integrable.sub' {f g : α → β} (hf : Integrable f μ) (hg : Integrable g μ) :
     Integrable (fun a ↦ f a - g a) μ := by simpa only [sub_eq_add_neg] using hf.add hg.neg
 
+@[fun_prop]
+theorem Integrable.enorm {f : α → ε} (hf : Integrable f μ) : Integrable (‖f ·‖ₑ) μ :=
+  ⟨hf.aestronglyMeasurable.enorm.aestronglyMeasurable, hf.hasFiniteIntegral.enorm⟩
+
 @[fun_prop]
 theorem Integrable.norm {f : α → β} (hf : Integrable f μ) : Integrable (fun a => ‖f a‖) μ :=
   ⟨hf.aestronglyMeasurable.norm, hf.hasFiniteIntegral.norm⟩
 
+-- TODO: generalise these to suitable enormed spaces: requires defining an enorm analogue
+-- of `HasSolidNorm` first
```

Comment:
`HasSolidNorm` doesn't make sense for `ENNReal`, but the conclusions do hold, so the lemmas would have to be changed entirely. I would remove the todo.